devstack/plugin.sh

```diff
@@ -41,9 +41,9 @@ function install_docker {
         sudo cat /lib/systemd/system/docker.service
         sudo sed -r -i "s|(ExecStart)=(.+)|\1=\2 --iptables=false|" /lib/systemd/system/docker.service
         sudo cat /lib/systemd/system/docker.service
-        sudo systemctl daemon-reload
-        sudo systemctl restart docker
-        sudo systemctl status docker
+        sudo systemctl --no-pager daemon-reload
+        sudo systemctl --no-pager restart docker
+        sudo systemctl --no-pager status docker
```

Comment:
The 'systemctl status' call was removed in PR #50, thats causing the merge conflict.